application/common/System.cpp

```diff
@@ -0,0 +1,152 @@
+#include "loom/script/loomscript.h"
+#include "loom/common/utils/utString.h"
+#include "loom/common/platform//platformThread.h"
+#include <iostream>
+#include <stdio.h>
+#include <stdlib.h>
+
+class System
```

Comment:
I think a class name like Process would be a lot more accurate than System. What do you think? @SmilyOrg @LukeLamothe thoughts?